src/components/ContentNav/index.tsx

```diff
@@ -109,40 +120,57 @@ export class ContentNav extends Component<ContentNavProps, ContentNavState> {
   render() {
     return (
       <React.Fragment>
-        <ul className="list" role="menu">
-          {this.state.items.map(({ name, subItems, expanded, active }: ItemProps) => {
-            return (
-              <Item
-                name={name}
-                subItems={subItems}
-                expanded={expanded}
-                active={active}
-                activate={this.itemClicked}
-                expand={this.expand}
-                key={name}
-              >
-                {subItems &&
-                  subItems.map(subItem => {
-                    return (
-                      <SubItem
-                        name={subItem.name}
-                        active={subItem.active}
-                        activate={this.subItemClicked}
-                        key={`${name}-${subItem.name}`}
-                      />
-                    );
-                  })}
-              </Item>
-            );
-          })}
-        </ul>
+        <div className="outer">
+          <ul className="list" role="menu">
+            {this.state.items.map(({ name, subItems, expanded, active }: ItemProps) => {
+              return (
+                <Item
+                  name={name}
+                  subItems={subItems}
+                  expanded={expanded}
+                  active={active}
+                  activate={this.itemClicked}
+                  expand={this.expand}
+                  key={name}
+                >
+                  {subItems.length > 0 &&
```

Comment:
It may be worth extracting this logic into it's own function, then you can just call `renderSubItems(subItems)` and will be a bit more readable.

Reply:
This has been updated.

Reply:
I like this change